tests/e2e/test_io_performance.py

```diff
@@ -35,22 +35,47 @@ def base_setup(self, request, interface_iterate, pod_factory):
 
     @pytest.mark.parametrize(
         argnames=[
-            "size", "io_direction", "jobs",
+            "size", "io_direction", "bs", "jobs",
             "runtime", "depth", "sheet_index"
         ],
         argvalues=[
             pytest.param(
-                *['1GB', 'rw', 1, 60, 4, 1],
+                *['1000GB', 'rw', '4k', 1, 1200, 4, 4],
                 marks=pytest.mark.polarion_id("OCS-676")
             ),
             pytest.param(
-                *['1GB', 'rw', 6, 60, 16, 2],
+                *['1000GB', 'rw', '4k', 4, 600, 8, 5],
                 marks=pytest.mark.polarion_id("OCS-677")
             ),
             pytest.param(
-                *['1GB', 'rw', 12, 60, 32, 3],
+                *['1000GB', 'rw', '4k', 8, 600, 8, 6],
                 marks=pytest.mark.polarion_id("OCS-678")
             ),
+            pytest.param(
+                *['1000GB', 'ro', '4k', 1, 1200, 8, 7]
+            ),
+            pytest.param(
+                *['1000GB', 'wo', '4k', 1, 1200, 8, 8]
+            ),
+            pytest.param(
+                *['1000GB', 'rw', '32k', 1, 1200, 8, 4]
+            ),
+            pytest.param(
+                *['1000GB', 'ro', '32k', 1, 1200, 8, 7]
+            ),
+            pytest.param(
+                *['1000GB', 'wo', '32k', 1, 1200, 8, 8]
+            ),
+            pytest.param(
+                *['1000GB', 'rw', '4m', 1, 1200, 8, 10]
+            ),
+            pytest.param(
+                *['1000GB', 'ro', '4m', 1, 1200, 8, 11]
+            ),
+            pytest.param(
+                *['1000GB', 'wo', '4m', 1, 1200, 8, 12]
+            ),
```

Comment:
Its good addition to suite. can we create/ask to create tc for the same. WDYT

Reply:
+1